addons/website_event/views/event_templates_page.xml

```diff
@@ -57,7 +57,10 @@
                     <span class="navbar-toggler-icon"></span>
                 </button>
                 <div id="o_wevent_event_submenu" class="collapse navbar-collapse">
-                    <ul class="navbar-nav flex-md-wrap w-100" t-att-data-menu_name="editable and 'Event Menu'" t-att-data-content_menu_id="editable and event.menu_id.id">
+                    <!-- For duplicated content menus (mainly used to have a different mobile UI),
+                        adding `content_menu_id` & `menu_name` attributes only once in the DOM is
+                        enough for the content menu to be detected and edited. -->
```

Comment:
IMO we don't need that comment @xO-Tx, it's just noise to document everything that is **not** necessary IMO :wink:

Reply:
OK, I'll remove it :+1: